palantirnet/federated-search-demo
# Base react

## Description

Allows running the base app at http://react.fs-demo.local

## Testing instructions

1. Checkout this branch
1. `vagrant reload --provision`
1. `vagrant ssh`
1. `phing react`
1. -- If you have not, build sites with `phing install-all` to populate the index.
1. Visit http://react.fs-demo.local and see content
